packages/angular/src/tracing.ts

```diff
@@ -49,6 +56,23 @@ export function routingInstrumentation(
 
 export const instrumentAngularRouting = routingInstrumentation;
 
+/**
+ * A custom BrowserTracing integration for Angular.
+ */
+export function browserTracingIntegration(
+  options?: Parameters<typeof originalBrowserTracingIntegration>[0],
+): Integration {
+  instrumentationInitialized = true;
+  hooksBasedInstrumentation = true;
+
+  return originalBrowserTracingIntegration({
+    ...options,
+    instrumentPageLoad: true,
+    // We handle this manually
+    instrumentNavigation: false,
+  });
+}
```

Comment:
I guess one downside of this approach I just thought of is that there will be no navigation span/txn at all, if users don't register the `TraceService`. Previously they would have gotten the default browser one. 

I think this is okay though - "incorrect" or incomplete SDK setup shouldn't be something we care too much about IMO. (Also, this is documented pretty clearly in docs)

Reply:
yeah, I think I will refactor this based on what we talked about that we actually never overwrite the defaults like this but only do it via checking if something else registered a hook instead!